openstudiocore/src/energyplus/ForwardTranslator/ForwardTranslateElectricEquipmentITEAirCooled.cpp

```diff
@@ -134,6 +134,9 @@ boost::optional<IdfObject> ForwardTranslator::translateElectricEquipmentITEAirCo
             LOG(Warn, modelObject.briefDescription() << " The FlowControlWithApproachTemperatures only applies to ITE zones with single duct VAV terminal unit.");
             return boost::none;
           }
+        } else {
+          LOG(Warn, modelObject.briefDescription() << " The FlowControlWithApproachTemperatures only applies to ITE zones with single duct VAV terminal unit.");
+          return boost::none;
         }
```

Comment:
This doesn't capture the case where there's not even a thermalZone to begin with.

Reply:
Proposed to address: https://github.com/NREL/OpenStudio/pull/3781/commits/4772146febe2cb873c03eed69ce2c32a880313cb